assets/src/scripts/monitoring-location/components/hydrograph/selectors/iv-data.js

```diff
@@ -0,0 +1,248 @@
+import {format} from 'd3-format';
```

Comment:
This essentially replaces drawing-data (which was poorly named). It follows a similar pattern to what we used in dailyValueHydrograph where selectors are provided the provide information to the rendering code as to how to draw and decorate the IV data.